### PR TITLE
Shuffleboard grid 2: electric boogaloo

### PR DESCRIPTION
**Pull request by greenblitz4590**

### CODE REVIEW

File: src/main/java/edu/greenblitz/tobyDetermined/subsystems/Console.java [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+import edu.wpi.first.wpilibj.DriverStation;
+import edu.wpi.first.wpilibj.shuffleboard.BuiltInLayouts;
+import edu.wpi.first.wpilibj.shuffleboard.ShuffleboardLayout;
+import edu.wpi.first.wpilibj.shuffleboard.ShuffleboardTab;
+
+public class Console {
+    private static ShuffleboardLayout shuffleboardConsole;
+
+    public static void log(String title, String message) {
+        title = title + " : " + DriverStation.getMatchTime();
```
Comment (Stormageddon37) [inline src/main/java/edu/greenblitz/tobyDetermined/subsystems/Console.java:12]: Consider using String.format

Comment (greenblitz4590) [reply src/main/java/edu/greenblitz/tobyDetermined/subsystems/Console.java:12]: consider it considered

File: src/main/java/edu/greenblitz/tobyDetermined/OI.java [hunk 2]
```diff
@@ -53,13 +53,18 @@ public static void disableHandling() {
 
 
 	private void initButtons() {
-		SwerveChassis.getInstance().setDefaultCommand(new CombineJoystickMovement(false));
+		SwerveChassis.getInstance().setDefaultCommand(new CombineJoystickMovement(true));
```
Comment (yonnbro) [inline src/main/java/edu/greenblitz/tobyDetermined/OI.java:56]: Sure?

**Review by yonnbro — APPROVED** [review]

File: src/main/java/edu/greenblitz/tobyDetermined/OI.java (hunk 2 above)
Comment (greenblitz4590) [reply src/main/java/edu/greenblitz/tobyDetermined/OI.java:56]: not really

**Pull request merged**